backend/votes/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from posts.models import Post
+from users.models import User
+from comments.models import Comment
+
+
+class Vote(models.Model):
+
+    class Vote(models.TextChoices):
```

Comment:
Iets van VoteEnum doen ofzo? 2 Classes met dezelfde naam, dat moet zowat niet goed gaat down the line zegmaar